README.markdown

```diff
@@ -22,14 +30,6 @@ Run application:
            edu.oregonstate.mist.dropwizardtest.DropwizardTestApplication \
            server configuration.yaml
 
-With New Relic:
```

Comment:
thanks for removing this. in the future i recommend using `git add -p` to break up commits into separate components. for instance, you would have separate commits for "remove new relic", "add configuration info", and "add link to ojdbc".